# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../aspell/gcc43.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- aspell-0.60.5/common/string.hpp.vanilla	2008-04-09 06:56:19.000000000 +0000
+++ aspell-0.60.5/common/string.hpp	2008-04-09 07:00:41.000000000 +0000
@@ -492,7 +492,7 @@
 
 namespace std
 {
-  template<> static inline void swap(acommon::String & x, acommon::String & y) {return x.swap(y);}
+  template<> inline void swap(acommon::String & x, acommon::String & y) {return x.swap(y);}
 }
 
 #endif
--- aspell-0.60.5/modules/speller/default/affix.hpp.vanilla	2008-04-09 07:09:04.000000000 +0000
+++ aspell-0.60.5/modules/speller/default/affix.hpp	2008-04-09 07:09:39.000000000 +0000
@@ -109,7 +109,7 @@
     }
     WordAff * expand_suffix(ParmString word, const unsigned char * new_aff,
                             ObjStack &, int limit = INT_MAX,
-                            unsigned char * new_aff = 0, WordAff * * * l = 0,
+                            unsigned char * new_aff2 = 0, WordAff * * * l = 0,
                             ParmString orig_word = 0) const;
     
   private:
